students/ZachCooper/session02/fizzbuzz.py

```diff
@@ -0,0 +1,9 @@
+for num in range(1, 101):
+	if num % 3 == 0 and num % 5 ==0:
```

Comment:
Looks like you're indenting quite a bit more than 4 spaces.  Might want to see if your editor is set up correctly to convert tabs to 4 spaces.